api/app/controllers/spree/api/v1/checkouts_controller.rb

```diff
@@ -25,6 +25,11 @@ def advance
         end
 
         def update
+          if @order.completed?
```

Comment:
@tommiller1 I think it will not raise exception but an unauthorised error with 401 status which is expected in this case as user is not authorised to update order after completion.